packages/react-conformance/src/defaultErrorMessages.tsx

```diff
@@ -0,0 +1,685 @@
+import { IsConformantOptions } from './types';
+import { ComponentDoc } from 'react-docgen-typescript';
+
+import chalk from 'chalk';
+import os from 'os';
+import * as _ from 'lodash';
+import * as path from 'path';
+import parseDocblock from './utils/parseDocblock';
+
+/* eslint-disable @typescript-eslint/naming-convention */
+
+// An array of all the failed isConformant tests for the component.
+const failedTests: string[] = [];
+
+export const defaultErrorMessages = {
+  'has-docblock': (componentInfo: ComponentDoc, testInfo: IsConformantOptions, error: string) => {
+    const { displayName, componentPath } = testInfo;
+    const fileName = path.basename(componentPath);
+    const docblock = parseDocblock(componentInfo.description);
+    const docBlockLength = _.words(docblock.description).length.toString();
+
+    // Message Description: Handles scenario where there is no existing docblock in the componentPath.
+    //
+    // It appears that "displayName" doesn't have a docblock in the file:: "componentPath".
+    // To Resolve this issue:
+    // 1. Consider adding a descriptive 5 to 25 word docblock in "fileName".
+    if (_.words(docblock.description).length === 0) {
+      console.log(
+        defaultErrorMessage(
+          `has-docblock`,
+          displayName,
+          'docblock in the file: ' + paragraph() + chalk.green.italic(componentPath),
+        ) +
+          resolveErrorMessages([
+            `Consider adding a descriptive` +
+              chalk.hex('#e00000')(' 5 ') +
+              'to' +
+              chalk.hex('#e00000')(' 25 ') +
+              'word docblock in ' +
+              chalk.hex('#e00000')(fileName),
+          ]),
+        receivedErrorMessage(error),
+      );
+    }
+
+    // Message Description: Handles scenario where a docblock is received but doesn't meet the min and max requirements.
+    //
+    // It appears that "displayName" doesn't have a docblock between 5 and 25 words.
+    // It has a word count of: "docBlockLength"
+    // Here is "displayName"'s docblock: "docblock.description"
+    // To Resolve this issue:
+    // 1. Make sure that your docblock meets the required word count of 5 to 25 words.
+    else {
+      console.log(
+        defaultErrorMessage(
+          `has-docblock`,
+          displayName,
+          'docblock between 5 and 25 words. It has a word count of: ' +
+            chalk.green.italic(docBlockLength) +
+            paragraph() +
+            'Here is ' +
```

Comment:
Not super important, but you might consider using template strings throughout the file to join some of these error messages together. (Template strings can also contain paragraph breaks.) For example:
```ts
        defaultErrorMessage(
          `has-docblock`,
          displayName,
          `docblock between 5 and 25 words. It has a word count of: ${chalk.green.italic(docBlockLength)}

Here is ${chalk.white(displayName)}'s docblock:

${chalk.green.italic(docblock.description)}`,
        )
```

Reply:
I added template strings but would prefer to keep the current paragraph function implementation.